introduction.md

```diff
@@ -0,0 +1,5 @@
+Here are the skills:
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?